doc/source/testing_pull_requests.rst

```diff
@@ -0,0 +1,178 @@
+      have multiple clones of different prs in the same folder
+
+.. note:: You can view the author/branch information looking at the
+          subtitle of the pull request, near the pull request status (expected
+          an `open` status)
+
+Using python-for-android commands directly from the pull request files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Enter inside the directory of the cloned repository in the above
+  step and run p4a command with proper args, eg::
+
+    .. codeblock:: bash
+
+        cd p4a-feature-fix-numpy
+        PYTHONPATH=. python3 -m pythonforandroid.toolchain apk \
+            --private=testapps/testapp_sqlite_openssl \
+            --dist-name=dist_test_app_python3_libs \
+            --package=org.kivy \
+            --name=test_app_python3_sqlite_openssl \
+            --version=0.1 \
+            --requirements=requests,peewee,sdl2,pyjnius,kivy,python3 \
+            --ndk-dir=/media/DEVEL/Android/android-ndk-r20 \
+            --sdk-dir=/media/DEVEL/Android/android-sdk-linux \
+            --android-api=27 \
+            --arch=arm64-v8a \
+            --permission=INTERNET \
+            --debug
+
+Things that you should know:
+
+
+    - The example above will build an testapp we will make use of the files of
+      the testapp named `testapp_sqlite_openssl.py` but we don't use the setup
+      file to build it so we must tell python-for-android what we want via
+      arguments
+    - be sure to at least edit the following arguments when running the above
+      command, since the default set in there it's unlikely that match your
+      installation:
+
+        - `--ndk-dir`: An absolute path to your android's NDK dir
+        - `--sdk-dir`: An absolute path to your android's SDK dir
+        - `--debug`: this one enables the debug mode of python-for-android,
+          which will show all log messages of the build. You can omit this one
+          but it's worth it to be mentioned, since this it's useful to us when
+          trying to find the source of the problem when things goes wrong
+    - The apk generated by the above command should be located at the root of
+      of the cloned repository, were you run the command to build the apk
+    - The testapps distributed with python-for-android are located at
+      `testapps` folder under the main folder project
+    - All the builds of python-for-android are located at
+      `~/.local/share/python-for-android`
+    - You should have a downloaded copy of the android's NDK and SDK
+
+Installing python-for-android using the github's branch of the pull request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Enter inside the directory of the cloned repository mentioned in
+  `Common steps` and install it via pip, eg.::
+
+    .. codeblock:: bash
+
+        cd p4a-feature-fix-numpy
+        pip3 install . --upgrade --user
```

Comment:
Minor: we can also suggest to use a virtualenv for not messing up with user ~/.local